packages/cli/src/hooks/analytics/eventUtils.ts

```diff
@@ -19,35 +25,22 @@ export function buildEvent(
 }
 
 export function buildError(arg: unknown) {
-  /**
-   * TODO: CDX-660: Make sure to remove any PII from the Error object.
-   *       error.message could contain data that is not allowed to be tracked for non-Trial users
-   *       example: orgID, sourceID, ...
-   */
-  if (arg instanceof Error) {
-    return arg;
-  }
+  const isErrorFromAPI = validate(arg, APIErrorSchema).valid;
```

Comment:
Extract this in a separate function:
```ts
  function isErrorFromAPI(arg: unknown): arg is APIErrorResponse {
    return validate(arg, APIErrorSchema).valid;
  }
```

Then, l31 instead of `if(isErrorFromAPI) { return new APIError(arg as APIErrorResponse); }` do:
`if(isErrorFromAPI(arg)) { return new APIError(arg); }`